src/helpers/wsClient/wsClient.ts

```diff
@@ -90,16 +136,31 @@ export class WebSocketClient {
   };
 
   constructor(options: WsOptions) {
```

Comment:
Trying to subscribe to any transaction without using specific ADM address. The handler is not triggered at all. @yoxira 

```js
import {AdamantApi, WebSocketClient} from '../../dist';

const nodes = ['https://ahead.adamant.im'];
const socket = new WebSocketClient({});

socket.on(transaction => {
  console.log('New transaction:', transaction.id, transaction.type);
});

const api = new AdamantApi({
  nodes,
  socket,
});
```

Reply:
Logs:

![image](https://github.com/user-attachments/assets/b9bf93a6-d34e-415a-bda9-5ffb51b67758)